src/routes/contact-requests.ts

```diff
@@ -0,0 +1,32 @@
+import express, { Request, Response, Router } from 'express';
+import { ContactRequestsCollection } from '../collections/contact-requests';
+
+const contactRepository: ContactRequestsCollection = new ContactRequestsCollection();
+contactRepository.initContactRequestData();
+
+const router: Router = express.Router();
+export default router;
```

Comment:
Here, I make an exception to my "never use default exports" rule, because this file _should only ever be imported in one place_ and it _should_ have naming conflicts with other similar files if you don't name it something useful. So... it gets a pass.

Reply:
I'm curious about your "never use default exports" rule. I do it all over the place in my Discord bot and would like to know why its not preferred.

Reply:
Using named exports forces the conceptual source of the code (the file it's written in) to conceive of a name for it. This usually makes for more descriptive names.

But the big reason is that if you import file A in files B, C, and D - they could each name it something different. This has two ill-effects:
1 - simply finding the usages of an exported _thing_ becomes more complicated. Not difficult, to be sure, but reducing friction in workflows is the name of the game here. Searching for a variable/function/class name and seeing it's usages, or ctrl-clicking the name for the same info + is infinitely easier than searching the file name, then seeing what each file has named it, then searching for that variable in each individual file for local usages; rinse, repeat.
2 - refactoring becomes much simpler as well. Mostly due to the above reason + the additional step of renaming all instances

So that's why I always use named exports (except in odd cases like the route one explained above)

Reply:
Gotcha! Makes sense to me. Thanks!